xilem/src/view/button.rs

```diff
@@ -35,24 +33,25 @@ where
         ctx.with_leaf_action_widget(|_| Pod::new(widget::Button::new(self.label.clone())))
     }
 
-    fn rebuild(
+    fn rebuild<'el>(
         &self,
         prev: &Self,
         _: &mut Self::ViewState,
         ctx: &mut ViewCtx,
-        mut element: WidgetMut<widget::Button>,
-    ) {
+        mut element: Mut<'el, Pod<widget::Button>>,
+    ) -> Mut<'el, Pod<widget::Button>> {
```

Comment:
The possible styles I can see for this are as follows. These are ordered by my preference:
1) `Mut<'el, Self::Element>`
1) `WidgetMut<'el, widget::Button>`
1) `Mut<'el, Pod<widget::Button>>`
1) `<Self::Element as ViewElement>::Mut<'el>`
1) `<Pod<widget::Button> as ViewElement>::Mut<'el>`

That is, I'd prefer for these to consistently use the `Mut<'el, Self::Element>` form; this matches the style seen in `build`, for example.

Reply:
Yeah too many possibilities for it..., I'm indifferent, I can change all of that to `Mut<'el, Self::Element>`.